### ISSUE TITLE
Alt text

### ISSUE THREAD

**Issue by GeopJr** [GeopJr/Tooth#3]
Tooth should be able to set alt text when posting media.

**Comment by GeopJr** [comment]
While a media upload redesign (and probably rewrite) is certainly needed, it's probably a good idea to have an anchor around that open's on the instance's post page.

The Mastodon post widget has many features that'd be close to impossible (at least for me) or too high-maintenance to implement like: "extract text from image" or the image "focus point" selector
